building/build-lambda-layers.sh

```diff
@@ -4,44 +4,74 @@ set -ex
 VERSION=$(poetry version --short)
 DIR_NAME=$(dirname "$PWD")
 
+PYTHON_VERSION=${1:-ALL}
+
 ARCH=$(arch)
 [ "${ARCH}" = "aarch64" ] && ARCH_SUFFIX="-arm64" # AWS Lambda, the name arm64 is used instead of aarch64
 
-echo "Building Lambda Layers for AWS SDK for pandas ${VERSION}"
+if [[ $PYTHON_VERSION == "ALL" ]]
```

Comment:
I wanted the ability to run this script without needing to build all 5 layers. Also, this will let us parallelize the process of building and testing layers in the future.